src/Parser/DemonHunter/Vengeance/Modules/Features/Checklist/Component.js

```diff
@@ -42,37 +44,102 @@ class VengeanceDemonHunterChecklist extends React.PureComponent {
           )}
         >
           <AbilityRequirement spell={SPELLS.IMMOLATION_AURA.id} />
+          <AbilityRequirement spell={SPELLS.SIGIL_OF_FLAME_CONCENTRATED.id} />
           {combatant.hasTalent(SPELLS.FRACTURE_TALENT.id) && <AbilityRequirement spell={SPELLS.FRACTURE_TALENT.id} />}
           {combatant.hasTalent(SPELLS.FELBLADE_TALENT.id) && <AbilityRequirement spell={SPELLS.FELBLADE_TALENT.id} />}
+          {combatant.hasTalent(SPELLS.FEL_DEVASTATION_TALENT.id) && <AbilityRequirement spell={SPELLS.FEL_DEVASTATION_TALENT.id} />}
         </Rule>
 
         <Rule
-          name="Maintain your buffs and debuffs"
+          name="Use your short defensive/healing cooldowns"
           description={(
             <React.Fragment>
-              <p>It is important to maintain these as they contribute a large amount to your DPS and HPS.</p>
+              <p>Use these to block damage spikes and keep damage smooth to reduce external healing required.</p>
               <a href="http://www.wowhead.com/vengeance-demon-hunter-rotation-guide#rotation-priority-list" target="_blank" rel="noopener noreferrer">More info.</a>
             </React.Fragment>
           )}
         >
           <AbilityRequirement spell={SPELLS.DEMON_SPIKES.id} />
-          <AbilityRequirement spell={SPELLS.SIGIL_OF_FLAME_CONCENTRATED.id} />
-        </Rule>
+          {combatant.hasTalent(SPELLS.SPIRIT_BOMB_TALENT.id) && !combatant.hasTalent(SPELLS.FEED_THE_DEMON_TALENT.id) &&(
+            <Requirement
+              name={(
+                <React.Fragment>
+                  <SpellLink id={SPELLS.SPIRIT_BOMB_TALENT.id} /> 4+ souls casts
+                </React.Fragment>
+              )}
+              thresholds={thresholds.spiritBombSoulConsume}
```

Comment:
Possible typo, you might have meant souls

Reply:
Technically EVERYTHING is using that. So i made a typo in its original creation ahaha